docs/services/sdk/error-handling.md

```diff
@@ -13,15 +13,39 @@ For situations where you do not want the invocation to be retried, but instead y
 to be propagated back to the caller, you can use a `terminal error`.
 
 ```typescript
-import { TerminalError } from "@restatedev/restate-sdk";
+import { TerminalError, ErrorCodes } from "@restatedev/restate-sdk";
 
 // Inside your handler throw terminal errors via:
-throw new TerminalError("Something went wrong.", { errorCode: 13, cause: "Something caused this." })
+throw new TerminalError("Something went wrong.", { errorCode: ErrorCodes.INTERNAL, cause: "Something caused this." })
```

Comment:
uh what do you do with the cause? We don't record that in the journal right?

Reply:
correct, it is not stored

Reply:
so then i would remove it, even from the API.

Reply:
it is part of the native JS error interface
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#rethrowing_an_error_with_a_cause
- can you file a issue for this and i will remove the mention of it in the docs in this pr

Reply:
Can we override that to disallow setting the cause?

Reply:
maybe, i am not sure

Reply:
Let's open an issue to verify this in the sdk

Reply:
https://github.com/restatedev/sdk-typescript/issues/207